<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BackButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>390</y>
      <width>51</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 99, 99);
border: 2px solid black;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>390</y>
      <width>51</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 99, 99);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>600</y>
      <width>1191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(78, 78, 78);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>1161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-image: url(:/images/wood.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>1161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(54, 54, 54);
border-bottom: 3px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="VodkaButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>550</y>
      <width>51</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>21</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-image: url(:/images/steel.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>871</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-image: url(:/images/steel.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>0</y>
      <width>21</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-image: url(:/images/steel.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="RumButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>550</y>
      <width>51</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="BurbonButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>550</y>
      <width>51</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="TequilaButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>550</y>
      <width>51</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CollinsGlassButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>51</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>Collins </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>640</y>
      <width>1161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(130, 130, 130);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="RocksGlassButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>51</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Rocks</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LimeButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>310</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="OliveButton">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>350</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Olives</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OrangeButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>330</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Oranges</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>370</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/steel.jpg);
border:2px solid black;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1211</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/bar.png);
background-size: 50% auto; </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>0</y>
      <width>151</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 255, 155);
border: 2px dotted black;
color: black;

</string>
    </property>
    <property name="text">
     <string>Drink algo here</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MartiniGlassButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Martini</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(97, 107, 101);</string>
    </property>
    <property name="text">
     <string>Wine Glass</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(97, 107, 101);</string>
    </property>
    <property name="text">
     <string>Wine Glass</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(97, 107, 101);</string>
    </property>
    <property name="text">
     <string>Wine Glass</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(97, 107, 101);</string>
    </property>
    <property name="text">
     <string>Wine Glass</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(97, 107, 101);</string>
    </property>
    <property name="text">
     <string>Wine Glass</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GinButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>550</y>
      <width>51</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="KahluaButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>550</y>
      <width>51</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>Kahlua</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OrangeLiqorButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>550</y>
      <width>51</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>OLiquor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>430</y>
      <width>731</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(230, 255, 246,100);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CopperMugButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Copper</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BittersButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>320</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Bitters</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>1201</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(230, 255, 246,100);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ShakertinButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>51</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Shaker</string>
    </property>
   </widget>
   <zorder>label_14</zorder>
   <zorder>label_13</zorder>
   <zorder>label</zorder>
   <zorder>label_9</zorder>
   <zorder>label_7</zorder>
   <zorder>label_2</zorder>
   <zorder>BackButton</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>VodkaButton</zorder>
   <zorder>label_8</zorder>
   <zorder>RumButton</zorder>
   <zorder>BurbonButton</zorder>
   <zorder>TequilaButton</zorder>
   <zorder>CollinsGlassButton</zorder>
   <zorder>RocksGlassButton</zorder>
   <zorder>LimeButton</zorder>
   <zorder>OliveButton</zorder>
   <zorder>OrangeButton</zorder>
   <zorder>label_11</zorder>
   <zorder>label_12</zorder>
   <zorder>MartiniGlassButton</zorder>
   <zorder>label_15</zorder>
   <zorder>label_16</zorder>
   <zorder>label_17</zorder>
   <zorder>label_18</zorder>
   <zorder>label_19</zorder>
   <zorder>GinButton</zorder>
   <zorder>KahluaButton</zorder>
   <zorder>OrangeLiqorButton</zorder>
   <zorder>label_10</zorder>
   <zorder>CopperMugButton</zorder>
   <zorder>BittersButton</zorder>
   <zorder>ShakertinButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1159</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
